packages/gatsby/src/schema/print.ts

```diff
@@ -1,17 +1,44 @@
-const fs = require(`fs-extra`)
-const {
+/* eslint-disable @typescript-eslint/no-use-before-define */
+
+import * as fs from "fs-extra"
+import {
   EnumTypeComposer,
   InputTypeComposer,
   InterfaceTypeComposer,
   ObjectTypeComposer,
   ScalarTypeComposer,
   UnionTypeComposer,
-} = require(`graphql-compose`)
-const report = require(`gatsby-cli/lib/reporter`)
+  SchemaComposer,
+  NamedTypeComposer,
+  ObjectTypeComposerFieldConfigMap,
+  ObjectTypeComposerFieldConfig,
+  ObjectTypeComposerArgumentConfigMap,
+  ObjectTypeComposerArgumentConfig,
+  Extensions,
+  isNamedTypeComposer,
+  EnumTypeComposerValueConfig,
+} from "graphql-compose"
+import report from "gatsby-cli/lib/reporter"
+
+import { internalExtensionNames } from "./extensions"
+import { GraphQLDirective } from "graphql"
 
-const { internalExtensionNames } = require(`./extensions`)
+const {
+  astFromValue,
+  print,
+  GraphQLString,
+  DEFAULT_DEPRECATION_REASON,
+} = require(`graphql`)
+const { printBlockString } = require(`graphql/language/blockString`)
+const _ = require(`lodash`)
```

Comment:
Any reason why you're not importing?

Reply:
Ah, I just moved moved requires we had in the middle of a file higher, but just didn't convert those, will do so

Reply:
Adjusted in [`e850e63` (#34787)](https://github.com/gatsbyjs/gatsby/pull/34787/commits/e850e63e1d2bfcca13456c877a4f8750bf8ee4bc)